NDB.Covid19/NDB.Covid19/Locales/so.json

```diff
@@ -88,22 +88,30 @@
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_TITLE": "Teknoloojiyada la adeegsaday",
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_CONTENT": "Barmaamijku wuxu isticmaalayaa Bluetooth waxaanu ku salaysanyahay teknoloojiyad ku jirta oo ay sameeyeen Apple iyo Google. Teknoloojiyadu waa nidaam la isku ogeysiiyo u dhawaanshaha qofka fayrusta qaba. Waad demin ama daari kartaa ogeysiinta haddii aad gasho nidaamka doorashooyinka ee telefoonkaaga.",
   "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_CONTENT2": "Telefoonkaagu wuxu samaynayaa Aqoonsi Bluetooth ah daqiiqadii 15-aad ba. Marka laba telefoon isu dhawaadaan, waxay is dhaafsanayaan Aqoonsiyada.",
-  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_CONTENT3": "Faahfaahin ku saabsan Smittestopp ka akhriso <a href=\"www.helsenorge.no/smittestopp\">www.helsenorge.no/smittestopp</a>\"",
+  "SETTINGS_PAGE_2_CONTENT_TEXT_PARAGRAPH_4_CONTENT3": "Faahfaahin ku saabsan Smittestopp ka akhriso <a href=\"https://www.helsenorge.no/smittestopp\">www.helsenorge.no/smittestopp</a>",
```

Comment:
Should navigate to the english page on helsenorge.
^change in both the excel sheet and this .json